phase1.py

```diff
@@ -0,0 +1,97 @@
+from model import vid_recognition as vr
```

Comment:
okay, ill rename our milestones and project pipeline into 3 phases instead of 2. good job man. I can demo you how the main.py (will rename it to phase2.py) and probably you can try to improve it too if there is any.

Reply:
@brebeek take a look at this file too, since this requires the skull detection model in use